### PR TITLE
Do not merge:  Passenger check in Python.

### PR DESCRIPTION
**Pull request by jomunoz**
Adding it it here for visibility purposes, we still do not know where the code should live.

### CODE REVIEW

File: python/passenger_status_check.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import shlex, subprocess
+
+# need to add this to the sudoers file:
+# dd-agent    ALL=(deploy) NOPASSWD: /bin/bash
+
+# passenger_check directory needs to be located in /opt/datadog-agent/agent/
+
+class PassengerStatusCheck(AgentCheck):
+    def check(self, instance):
+        args = shlex.split(instance.get('command'))
```
Comment (jomunoz) [inline python/passenger_status_check.py]: Sorry, I don't understand.

Comment (davexunit) [inline python/passenger_status_check.py]: We're leaking implementation details into the configuration file.  We will always be calling `passenger-status` with the `--show=xml` flag.  I think you are providing too much customization.  We should allow the executable file name to be tweaked, with a default of simply `passenger-status`, and nothing more.

Comment (jomunoz) [inline python/passenger_status_check.py]: I see, I found that the check is being run by a user named `dd-agent`. So it is not root the one executing the check, hence to make it work I added `sudo -i -u deploy rvmsudo passenger-status --show=xml` and the entry in the sudoers file. 
But now that you pointed out the customization, maybe I can have it run without going thru the deploy user.

Comment (davexunit) [inline python/passenger_status_check.py]: How about just leaving off the flags then?

File: python/passenger_status_check.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import subprocess
+
+# need to add this to the sudoers file:
+# dd-agent    ALL=(deploy) NOPASSWD: /bin/bash
+
+# passenger_check directory needs to be located in /opt/datadog-agent/agent/
+
+class PassengerStatusCheck(AgentCheck):
+    def check(self, instance):
+        xml_data = subprocess.Popen(['passenger-status', '--show=xml'], stdout=subprocess.PIPE).communicate()[0]
```
Comment (davexunit) [inline python/passenger_status_check.py]: Now `passenger-status` is a hardcoded string.  It should still be configurable, but with `passenger-status` as the default.

**Pull request closed without merging**